### PR TITLE
Nicer README.md

### PR DESCRIPTION
**Pull request by inson1**
- [x] I read the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).

github has bug, so see readme here https://github.com/inson1/Gallery/blob/patch-3/README.md

May I add it do all repositories?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
-# Fossify Gallery
+<div align="center">
 
 <img alt="Logo" src="graphics/icon.webp" width="120" />
 
-<a href="https://f-droid.org/en/packages/org.fossify.gallery/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-en.svg" alt="Get it on F-Droid" height=80/></a> <a href="https://apt.izzysoft.de/fdroid/index/apk/org.fossify.gallery"><img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png" alt="Get it on IzzyOnDroid" height=80/></a>
+# Fossify Gallery
+
+### Browse your memories without any interruptions with this photo and video gallery
```
Comment (Aga-C) [inline README.md:7]: Don't use headers for formatting, they have a semantic value. Also, you shouldn't jump in headers hierarchy like this. After the first level comes the second level etc.

If you want to read more: https://www.wearediagram.com/blog/how-not-to-use-html-headings (it's about HTML not Markdown, but it gets transformed to HTML anyway).

**Comment by inson1** [comment]
@Aga-C so what should I use for text size formatting if css doesnt work and I cant use headers?

**Comment by Aga-C** [comment]
> so what should I use for text size formatting if css doesnt work and I cant use headers?

Markdown is meant to be simple, so you only have these: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text. Also, basic HTML styling (without CSS) also works here, you even used center-aligned div.

**Comment by naveensingh** [comment]
@inson1 When all apps are released everywhere, all READMEs will be reformatted for example like [this](https://github.com/you-apps/RecordYou) or [this](https://github.com/TeamNewPipe/NewPipe). No need to worry about the README unless it's about adding a very important link.

**Pull request closed without merging**